cmd/drone-docker/main.go

```diff
@@ -324,6 +335,8 @@ func run(c *cli.Context) error {
 			Quiet:       c.Bool("quiet"),
 			Platform:    c.String("platform"),
 			SSHAgentKey: c.String("ssh-agent-key"),
+			ImportCache: decodeImportCache(c.String("import-cache")),
```

Comment:
Create a custom variable instead of StringFlag. https://github.com/drone-runners/drone-runner-aws/pull/237/files#diff-cf7fbb42c66fce9a98485449d21136aba758d0974fabd1c18a03dff5749dcc23R12

Reply:
Done